core-versions-fe/src/core-versions/mockVersions.ts

```diff
@@ -1,14 +1,63 @@
-export async function mockVersions({count = 15}): Promise<any> {
```

Comment:
what is the difference between the mockVersions.ts and mock-versions.ts files? can one of them be deleted?

Reply:
Yeah, the mock-versions.ts is an old file that I had renamed mockVersions.ts and then edited, not sure why it's still in there, mock-can be deleted

Reply:
please remove mock-versions.ts if redundant.